MapleServer2/PacketHandlers/Game/BlackMarketHandler.cs

```diff
@@ -240,4 +262,34 @@ private static void HandlePurchase(GameSession session, PacketReader packet)
         MailHelper.BlackMarketTransaction(purchasedItem, listing, session.Player.CharacterId, listing.Price, removeListing);
         session.Send(BlackMarketPacket.Purchase(listingId, amount));
     }
+
+    private static ItemStat ReadStat(int statId, int value)
+    {
+        // Normal Stat with percent value
+        if (statId >= 1000 && statId < 11000)
+        {
+            NormalStat normalStat = new NormalStat();
+            float percent = (float) Math.Round(value * 0.0001f + 0.0005f, 4);
```

Comment:
why is this needed again? Can't you just cast to float the value?

Reply:
Without rounding it, I get values like
`Float Value: 0.0009999999`